man/systemd.service.xml

```diff
@@ -1489,11 +1495,6 @@ ExecStart=/bin/echo $ONE $TWO $THREE</programlisting>
     includes e.g. <varname>$USER</varname>, but not
     <varname>$TERM</varname>).</para>
 
-    <para>Note that shell command lines are not directly supported. If
-    shell command lines are to be used, they need to be passed
-    explicitly to a shell implementation of some kind. Example:</para>
-    <programlisting>ExecStart=sh -c 'dmesg | tac'</programlisting>
```

Comment:
i think we should keep this in some form, just to emphasize things. of course should be updated to point people to `|` instead of `sh -c`